db-colp/bash/04_export.sh

```diff
@@ -1,40 +1,39 @@
 #!/bin/bash
-source bash/config.sh
+source ../bash/utils.sh
+set_env ../.env
 
 rm -rf output
 mkdir -p output && (
     cd output
 
     echo "Exporting COLP"
-    CSV_export colp
-    SHP_export colp POINT
-    FGDB_export colp POINT
+    csv_export colp
+    csv_export colp POINT
```

Comment:
is this intended to be csv and not shp?

Reply:
Ooh thank you, that's not a great little change